server/lib/device/device.notify.js

```diff
@@ -0,0 +1,44 @@
+const get = require('get-value');
+
+const logger = require('../../utils/logger');
+const { EVENTS } = require('../../utils/constants');
+
+const FUNC_BY_EVENT = {
+  [EVENTS.DEVICE.CREATE]: 'onNewDevice',
+  [EVENTS.DEVICE.UPDATE]: 'onUpdateDevice',
+  [EVENTS.DEVICE.DELETE]: 'onDeleteDevice',
+};
+
+/**
+ * @description Notify a device is created.
+ * @param {Object} device - Created device.
+ * @param {string} event - Event to send.
+ * @example
+ * device.notify({ service_id: 'a810b8db-6d04-4697-bed3-c4b72c996279'}, 'device.create');
+ */
+function notify(device, event) {
+  logger.debug(`Notify device ${device.selector} creation`);
+
+  const serviceFuncName = FUNC_BY_EVENT[event];
+
+  if (!serviceFuncName) {
+    logger.warn(`Event ${event} not handled to notify device changes`);
+  } else {
+    // send global event
+    this.eventManager.emit(event, device);
+
+    // notify concerned service
+    const service = this.serviceManager.getServiceById(device.service_id);
+    if (service === null) {
+      logger.warn(`Service ${device.service_id} was not found.`);
+    } else if (typeof get(service, `device.${serviceFuncName}`) !== 'function') {
+      logger.warn(`Function device.${serviceFuncName} in service ${service.name} does not exist.`);
```

Comment:
I think a warn is too high, people are scared in the forum when they see a warning in their logs ! 

A simple "info" might be enough IMO

Reply:
Ok, I will :)

Reply:
I think we should keep 2 others in warn level, which could reveals bugs.